beacon_node/network/src/network_beacon_processor/rpc_methods.rs

```diff
@@ -1154,33 +1154,42 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
 
         let request_start_slot = Slot::from(req.start_slot);
 
-        let data_availability_boundary_slot = match self.chain.data_availability_boundary() {
-            Some(boundary) => boundary.start_slot(T::EthSpec::slots_per_epoch()),
-            None => {
-                debug!("Deneb fork is disabled");
-                return Err((RpcErrorResponse::InvalidRequest, "Deneb fork is disabled"));
-            }
-        };
+        let column_data_availability_boundary_slot =
+            match self.chain.column_data_availability_boundary() {
+                Some(boundary) => boundary.start_slot(T::EthSpec::slots_per_epoch()),
+                None => {
+                    debug!("Deneb fork is disabled");
+                    return Err((RpcErrorResponse::InvalidRequest, "Deneb fork is disabled"));
```

Comment:
It could be Fulu or Deneb that is disabled, but I think it's not wrong to say Fulu is disabled (because Deneb being disabled also implies Fulu being disabled).

```suggestion
                    debug!("Fulu fork is disabled");
                    return Err((RpcErrorResponse::InvalidRequest, "Fulu fork is disabled"));
```